crates/containerd-shim-wasmedge/src/executor.rs

```diff
@@ -51,14 +42,34 @@ impl Executor for WasmEdgeExecutor {
             .ok_or_else(|| anyhow::Error::msg("Not found wasi module"))
             .map_err(|err| ExecutorError::Execution(err.into()))?;
 
+        let args = oci::get_module_args(spec);
+        let mut module_args = None;
+        if !args.is_empty() {
+            module_args = Some(args.iter().map(|s| s as &str).collect())
+        }
+
+        debug!("module args: {:?}", module_args);
```

Comment:
At some point we need to go through the codebase and look at the logging, because I'm worried that someone might pass secrets as environment variables or module arguments and we'll accidentally log them. It's not a problem yet, just something to keep in mind.

Reply:
yes, we might want to take a more structure approach to logging as well.

I can drop some of these debug statements.  It was helpful when figuring this out but your concern is valid.